Units/MMLAddon/LPInc/Classes/MML/lptmufasabitmap.pas

```diff
@@ -512,25 +512,25 @@ procedure Register_TMufasaBitmap(Compiler: TLapeCompiler);
     addClassVar('TMufasaBitmap', 'Height', 'Integer', @TMufasaBitmap_Height_Read);
     addGlobalFunc('procedure TMufasaBitmap.SetPersistentMemory(mem: PtrUInt; awidth, aheight: integer);', @TMufasaBitmap_SetPersistentMemory);
     addGlobalFunc('procedure TMufasaBitmap.ResetPersistentMemory();', @TMufasaBitmap_ResetPersistentMemory);
-    addGlobalFunc('function TMufasaBitmap.PointInBitmap(x,y : integer): boolean;', @TMufasaBitmap_PointInBitmap);
-    addGlobalFunc('procedure TMufasaBitmap.ValidatePoint(x,y : integer);', @TMufasaBitmap_ValidatePoint);
+    addGlobalFunc('function TMufasaBitmap.PointInBitmap(x,y : integer): boolean; constref;', @TMufasaBitmap_PointInBitmap);
+    addGlobalFunc('procedure TMufasaBitmap.ValidatePoint(x,y : integer); constref;', @TMufasaBitmap_ValidatePoint);
     addGlobalFunc('function TMufasaBitmap.SaveToFile(const FileName : string): boolean;', @TMufasaBitmap_SaveToFile);
```

Comment:
Yeah, I was VERY quick with TMufasaBitmap, just took the ones that I instantly recognized.